host-controller/src/main/java/org/jboss/as/domain/controller/operations/coordination/ServerOperationResolver.java

```diff
@@ -257,7 +258,7 @@ private static ModelNode cleanOpForDontPropagate(ModelNode op) {
                     return Collections.singletonMap(allServers, operation);
                 }
                 case DEPLOYMENT: {
-                    return getServerExplodedDeploymentOperations(operation, address, domain, host);
+                    return getServerExplodedDeploymentOperations(getOriginalDomainModel(context), operation, address, host);
```

Comment:
just a note to myself, `getOriginalDomainModel(context)` replaces the `domain` parameter (order of params has changed)